layer/layer_utils.h

```diff
@@ -15,11 +15,4 @@
 #ifndef STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_UTILS_H_
 #define STADIA_OPEN_SOURCE_PERFORMANCE_LAYERS_UTILS_H_
```

Comment:
This header is empty now, but I'll layer macros here in a next PR. I didn't remove it entirely because of that.